automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/analysis/MethodConfigSpec.scala

```diff
@@ -245,28 +233,6 @@ class MethodConfigSpec extends FreeSpec with Matchers with LazyLogging
     }
   }
 
-  "launch a method config from the method repo" in withWebDriver { implicit driver =>
```

Comment:
removed this test because it performs identical steps as in test `launch and delete a simple workflow` before the action delete.